### ISSUE TITLE
USB CDC Transmit works for either port, but won't work for both simultaneously

### ISSUE THREAD

**Issue by stephenhensley** [electro-smith/libDaisy#160]
e.g.
this doesn't work (only the first one called actually transmits):
```C++
dsy_system_delay(500);
sprintf(buff, "Tick:\t%d\r\n", hello_count);
hw.usb_handle.TransmitExternal((uint8_t*)buff, strlen(buff));
sprintf(buff, "Tick:\t%d\r\n", hello_count);
hw.usb_handle.TransmitInternal((uint8_t*)buff, strlen(buff));
hello_count = (hello_count + 1) % 100;
```
while either of these work fine on:
```C++
dsy_system_delay(500);
sprintf(buff, "Tick:\t%d\r\n", hello_count);
hw.usb_handle.TransmitInternal((uint8_t*)buff, strlen(buff));
hello_count = (hello_count + 1) % 100;
```

```C++
dsy_system_delay(500);
sprintf(buff, "Tick:\t%d\r\n", hello_count);
hw.usb_handle.TransmitExternal((uint8_t*)buff, strlen(buff));
hello_count = (hello_count + 1) % 100;
```

the redundant `sprintf` is there because at first I thought the transmit function may have been consuming the buffer when transmitting, but it doesn't.

**Comment by jcolicchio** [comment]
Issue is in `usbd_cdc.c`, `h_cdc_fs` or `h_cdc_hs` is being used by both USB ports. We can declare a single `h_cdc[256]` and use `pdev->id` to select which handle to use during init, or we could declare `h_cdc[256]` as an array of `USBD_CDC_HandleTypeDef *`, and malloc/free handles on `Init` and `DeInit`

I can draft a PR with one of these fixes for review and iteration

**Comment by stephenhensley** [comment]
fixed in #213
